pkg/testgen/configurers.go

```diff
@@ -0,0 +1,131 @@
+/*
+Copyright 2022 The cert-manager Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package testgen
```

Comment:
Same with this file: sed replace `test` with `prow`. Maybe being pedantic but to me, test generation sounds like generating unit test data or e2e tests, where as prow config generation is much more clear.